### PR TITLE
I18n sync In & Up 11/04

### PR DESCRIPTION
**Pull request by deploy-code-org**

### CODE REVIEW

File: i18n/locales/source/dashboard/base.yml [hunk 1]
```diff
@@ -41,10 +41,11 @@ en:
     attributes:
       user:
         name: 'Display Name'
-        name_example: 'Display Name<br/>(e.g. Cool Coder or Jane S.)'
+        name_example: 'Display Name(e.g. Cool Coder or Jane S.)'
```
Comment (Hamms) [inline i18n/locales/source/dashboard/base.yml]: @daynew have you confirmed that this change doesn't result in a visual change on the signup page?

**Review by Hamms — APPROVED** [review]

File: i18n/locales/source/course_content/Hour of Code/dance-2019.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    "authored_hints": {
+      "validation": "**How do I pass this level?**\n\n-   Just press the \"Finish\" button!",
+      "repeat_every_ideas": "**Need some ideas?**\n\n-   Repeatedly change a dancer's size to make it grow.\n-   Repeatedly change a dancer's rotation to make it spin.\n-   Repeatedly change a dancer's horizontal or vertical position to make it move across the screen.\n\n_You can also use negative numbers to create an opposite effect!_",
+      "next_previous": "**Optional Challenge**\n\n-   You can also use this new event block to cycle through a list of dance moves by setting the move to `(Random)`, `(Next)`, or `(Previous)`.\n    <xml><block type=\"Dancelab_changeMoveEachLR\">\n    <title name=\"GROUP\" config=\"sprites, &quot;BEAR&quot;, &quot;CAT&quot;, &quot;DOG&quot;, &quot;DUCK&quot;, &quot;FROG&quot;, &quot;PINEAPPLE&quot;, &quot;ROBOT&quot;, &quot;SHARK&quot;, &quot;UNICORN&quot;\">???</title>\n    <title name=\"MOVE\">\"rand\"</title>\n    <title name=\"DIR\">-1</title>\n    </block></xml>"
```
Comment (Hamms) [inline i18n/locales/source/course_content/Hour of Code/dance-2019.json:89]: why was this blockly XML not redacted?

File: i18n/locales/source/course_content/2019/dance-extras-2019.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
```
Comment (Hamms) [inline i18n/locales/source/course_content/2019/dance-extras-2019.json:1]: I'd expect this file to be added under `Hour of Code`, not under the `2019` directory; do we know why it's here? Is it supposed to be?

Comment (nkiruka) [reply i18n/locales/source/course_content/2019/dance-extras-2019.json:1]: Good question!  I'm not sure.  I will check with the starlabs team.

**Pull request merged**